generators/docker-compose/prompts.js

```diff
@@ -287,7 +287,8 @@ function getAppFolders(input, applicationType) {
                     if ((fileData['generator-jhipster'].baseName !== undefined)
                         && ((applicationType === undefined)
                             || (applicationType === fileData['generator-jhipster'].applicationType)
-                            || ((applicationType === 'microservice') && ('gateway' === fileData['generator-jhipster'].applicationType)))) {
+                            || ((applicationType === 'microservice') && ('gateway' === fileData['generator-jhipster'].applicationType))
```

Comment:
Oh I see, it's a little bit confusing. This `applicationType` is a parameter in `function getAppFolders(input, applicationType) {`, and his value can be: monolith or microservice, accordingly to the first question: monolith or microservice